pkgs/development/ocaml-modules/cduce/types.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildDunePackage, num }:
+
+let
+  common = import ./common.nix {
+    inherit lib;
+  };
+in
+buildDunePackage rec {
+  pname = "cduce-types";
+
+  minimalOCamlVersion = "4.04.2";
```

Comment:
opam file say 4.07.0 but it can be build on lower bound so I just put the lower bound that compile.